src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
+$day_size: 100px;
+$gap_size: 1px;
+$padding_size: 10px;
+$starting_days: (1 mon, 2 tue, 3 wed, 4 thu, 5 fri, 6 sat, 7 sun);
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: $gap_size;
+  width: 7 * $day_size + 6 * $gap_size + 2 * $padding_size;
```

Comment:
The calculation for the width of the `.calendar` class seems incorrect. The expression `7 * $day_size + 6 * $gap_size + 2 * $padding_size` should be enclosed in parentheses to ensure the correct order of operations. It should be `(7 * $day_size) + (6 * $gap_size) + (2 * $padding_size)` to avoid any potential issues with operator precedence.